### ISSUE TITLE
Ensure text remains visible during webfont load

### ISSUE THREAD

**Issue by harrisonpim** [harrisonpim/theblackhart.co.uk#1]
https://web.dev/font-display/?utm_source=lighthouse&utm_medium=unknown
